homeassistant/components/media_player/emby.py

```diff
@@ -0,0 +1,296 @@
+"""
+Support to interface with the Emby API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.emby/
+"""
+import logging
+
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.media_player import (
+    MEDIA_TYPE_TVSHOW, MEDIA_TYPE_VIDEO, SUPPORT_NEXT_TRACK, SUPPORT_PAUSE,
+    SUPPORT_SEEK, SUPPORT_STOP, SUPPORT_PREVIOUS_TRACK, MediaPlayerDevice,
+    PLATFORM_SCHEMA)
+from homeassistant.const import (
+    CONF_HOST, CONF_API_KEY, CONF_PORT, DEVICE_DEFAULT_NAME, STATE_IDLE,
+    STATE_OFF, STATE_PAUSED, STATE_PLAYING, STATE_UNKNOWN)
+from homeassistant.helpers.event import (track_utc_time_change)
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pyemby==0.1']
+
+MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
+MIN_TIME_BETWEEN_FORCED_SCANS = timedelta(seconds=1)
+
+DEFAULT_PORT = 8096
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_EMBY = SUPPORT_PAUSE | SUPPORT_PREVIOUS_TRACK | SUPPORT_NEXT_TRACK | \
+    SUPPORT_STOP | SUPPORT_SEEK
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
```

Comment:
I guess that defaulting to `localhost` and make `host` optional would be an option to make the setup simpler.